.travis.yml

```diff
@@ -1 +1,7 @@
-language: java
\ No newline at end of file
+dist: xenial
+language: java
+jdk:
+  - openjdk8
+env:
+  - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
```

Comment:
Doesn't this work?
PATH="$JAVA_HOME/bin:$PATH"

Reply:
Done. It worked.